agent_build_refactored/scripts/filter_matrix_jobs.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2014-2022 Scalyr Inc.
```

Comment:
This basically just a part of the `agent_build_refactored/scripts/get_github_actions_job_matrices.py` file, which was logically split into three files.